### DISCUSSION TITLE
Support React-native: use react-native-gcanvas to work pixi.js

### DISCUSSION THREAD

**Discussion by Timson020** [pixijs/pixijs#8650] [General]
hey, guys.
I try to use react-native-gcanvas to work with pixi.js

It can't work in android, but can work in ios

can help me ? 

thank you a lot 

[pixi && gcanvas demo](https://github.com/Timson020/react-native-demo)

**Comment by Timson020** [comment]
not error message, but not anything render on android.

can work on ios

**Comment by flyskywhy** [comment]
Fixed in https://github.com/pixijs/pixijs/issues/5723#issuecomment-1411277994
